hail/src/main/scala/is/hail/rvd/RVD.scala

```diff
@@ -1534,3 +1534,13 @@ object RVD {
     fileData
   }
 }
+
+case class BroadcastRVD(@transient val backend: SparkBackend, @transient val rvd: RVD) extends Serializable {
```

Comment:
It looks to me like these don't need to be `val`s at all (and this doesn't need to be a case class), which should avoid the need for `transient`. In general, I agree with Cotton that we make too many things case classes.

Reply:
fair.